controller/volume_controller.go

```diff
@@ -4343,6 +4346,12 @@ func (c *VolumeController) IsReplicaUnavailable(r *longhorn.Replica) (bool, erro
 		return true, nil
 	}
 
+	if isRWX, _ := c.ds.IsRegularRWXVolume(r.Spec.VolumeName); isRWX {
+		if isDelinquent, _ := c.ds.IsNodeDelinquent(r.Spec.NodeID, r.Spec.VolumeName); isDelinquent {
+			return true, nil
+		}
+	}
```

Comment:
Need to handle the error returned

Reply:
Yeah, looks like it is redundant. Removed it